changelogs/v0.25.0.md

```diff
@@ -0,0 +1,131 @@
+# v0.25.0 Changes
+
+### Breaking
+
+Changes that are expected to cause an incompatibility.
+
+* **config**: remove support for base64 encoded certificates in the [`certificates`](https://www.pomerium.com/docs/reference/certificates#certificates) field. It may only contain file locations. See https://github.com/pomerium/pomerium/pull/4718 by @calebdoxsey for details.
+* **config**: remove [`debug`](https://www.pomerium.com/docs/reference/debug#summary) option, always use json logs by @calebdoxsey in https://github.com/pomerium/pomerium/pull/4857
+
+### New
+* **authenticate**: Refactoring identity authenticators to initiate redirect. For AWS Cognito, please allow the following sign out `https://{AUTHENTICATE_DOMAIN}/.pomerium/signed-out` URL. See more details in https://github.com/pomerium/pomerium/pull/4858 by @calebdoxsey.
```

Comment:
@ZPain8464: we should call out the "Allowed sign-out URLs" requirement in the https://www.pomerium.com/docs/identity-providers/cognito#setting-up-aws-cognito guide as well.

Reply:
@kenjenkins noted. I'll create a follow-up issue to update this.

Reply:
This was added to docs in this PR: https://github.com/pomerium/documentation/pull/1138